kamailio/trunks/config/kamailio.cfg

```diff
@@ -978,71 +978,70 @@ route[SELECT_NEXT_GW] {
 route[PARSE_X_HEADERS] {
     # Extract routing tag
     $var(header) = 'X-Info-RoutingTag';
-    route(PARSE_OPTIONAL_X_HEADER);
+    $var(header-value) = $hdr($var(header));
```

Comment:
$var(header) is not used anymore, isnt it?

```$var(header-value) = $hdr('X-Info-RoutingTag');```

Reply:
Evaluate second commit carefully (I will squash later if needed), check for errors please.